css/index.css

```diff
@@ -59,3 +65,32 @@ em {
     color: #f33;
 }
 
+#info{
+    z-index: 1;
+    background: rgba(255,255,255,.9);
+    position: absolute;
+    width: 100%;
+    height: auto;
+    left: -100%;
+}
+
+#info-wrapper{
+    padding: 75px;
+}
+
+button:focus{outline: none;}
+
+button{
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 400;
+    border: 1px solid #191919;
+    background: transparent;
+    text-transform: uppercase;
+    letter-spacing: 1px;
+    padding: 15px;
+}
+
+#info p{
+    line-height: 1.5;
+    font-weight: 400;
+}
```

Comment:
[Add newline at the end of files to be POSIX compliant.](http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)